Week1/homework/js-exercises/arrayAnimals.js

```diff
@@ -0,0 +1,7 @@
+var items =  [];
+console.log("The value of array is undefined");
+console.log(items);
+var animals = ["horse","donkey","zebra"];
+console.log(animals[1]);
+animals.unshift("piglet");
+console.log(animals[1]);
```

Comment:
Aahaan. I am not sure if you are also reading some material online regarding JavaScript topics but if you are, I think those materials are a bit outdated since you are using `var` keyword instead of `let` or `const`. Even though using `var` is correct but we should strive for creating variables now with the new ES6 features `let` and `const`.

For example, `const` here would be a much better option than `let` or `var`.